src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,338 @@
+package duke;
+import static duke.aesthetics.Emojis.*;
```

Comment:
I think imported class should be listed explicitly. Please refer to the coding standard.